[2023-03-22T09:39:05.364+0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_gcp_12.bigquery_external_table_task scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T09:39:05.367+0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_gcp_12.bigquery_external_table_task scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T09:39:05.367+0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T09:39:05.367+0400] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-03-22T09:39:05.367+0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T09:39:05.379+0400] {taskinstance.py:1300} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2023-03-21 00:00:00+00:00
[2023-03-22T09:39:05.382+0400] {standard_task_runner.py:55} INFO - Started process 676977 to run task
[2023-03-22T09:39:05.383+0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'data_gcp_12', 'bigquery_external_table_task', 'scheduled__2023-03-21T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/data_gsp.py', '--cfg-path', '/tmp/tmpxpqwkdyd']
[2023-03-22T09:39:05.384+0400] {standard_task_runner.py:83} INFO - Job 94: Subtask bigquery_external_table_task
[2023-03-22T09:39:05.408+0400] {task_command.py:388} INFO - Running <TaskInstance: data_gcp_12.bigquery_external_table_task scheduled__2023-03-21T00:00:00+00:00 [running]> on host civispc
[2023-03-22T09:39:05.447+0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_gcp_12
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-21T00:00:00+00:00
[2023-03-22T09:39:05.449+0400] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-03-22T09:39:05.449+0400] {credentials_provider.py:323} INFO - Getting connection using `google.auth.default()` since no key file is defined for hook.
[2023-03-22T09:39:05.993+0400] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=data_gcp_12, task_id=bigquery_external_table_task, execution_date=20230321T000000, start_date=20230322T053905, end_date=20230322T053905
[2023-03-22T09:39:06.036+0400] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-22T09:39:06.056+0400] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
